example/listenvironments/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/oauth2"
+)
+
+// An example of how to use ListEnvironments method with EnvironmentListOptions.
+// It's runnable with the following command:
+// export GITHUB_TOKEN=your_token export GITHUB_REPOSITORY_OWNER=your_owner export GITHUB_REPOSITORY_NAME=your_repo && go run .
```

Comment:
Please move these lines immediately before the `package main` above (but separated from the initial comment required in this repo):

```go
// Copyright 2022 The go-github AUTHORS. All rights reserved.
//
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// listenvironments is an example of how to use the ListEnvironments method with EnvironmentListOptions.
// It's runnable with the following command:
//     export GITHUB_TOKEN=your_token
//     export GITHUB_REPOSITORY_OWNER=your_owner
//     export GITHUB_REPOSITORY_NAME=your_repo
//     go run .
package main
```

Reply:
Done ✅